app/services/hyrax/tika_file_characterization_service.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+# Class for Apache Tika based file characterization service
+# defines the Apache Tika based characterization service a ValkyrieFileCharacterization service
+# @since 3.0.0
+module Hyrax
+  class TikaFileCharacterizationService
+    attr_reader :file_node, :persister
+    def initialize(file_node:, persister:)
+      @file_set = file_node
+      @persister = persister
+    end
+
+    # characterizes the file_node passed into this service
+    # Default options are:
+    #   save: true
+    # @param save [Boolean] should the persister save the file_node after Characterization
+    # @return [FileNode]
+    # @example characterize a file and persist the changes by default
+    #   Valkyrie::FileCharacterizationService.for(file_node, persister).characterize
+    # @example characterize a file and do not persist the changes
+    #   Valkyrie::FileCharacterizationService.for(file_node, persister).characterize(save: false)
```

Comment:
I don't see where you accepting this `save` argument. Should we remove this documentation?